src/EchoStateNetworks.jl

```diff
@@ -36,6 +37,7 @@ mutable struct EchoStateNetwork{T<:AbstractFloat}
         leaking_rate::AbstractFloat=1.0, 
         teacher_forcing::Bool=true,
         rng::AbstractRNG=MersenneTwister(rand(UInt32)),
+        activation::Function=x->tanh(x),
```

Comment:
It can be written without using anonymous functions.
```suggestion
        activation::Function=tanh,
```